internal/backend/sessions/sessions.go

```diff
@@ -118,6 +119,8 @@ func (s *sessions) Start(ctx context.Context, session sdktypes.Session) (sdktype
 	ctx = akCtx.WithRequestOrginator(ctx, akCtx.SessionWorkflow)
 	ctx = akCtx.WithOwnershipOf(ctx, s.svcs.DB.GetOwnership, session.EnvID().UUIDValue())
 
+	// REVIEW: pass session by pointer to DB.CreateSession and update createdAT?
+	session = session.WithCreatedAndUpdatedAt(time.Now())
```

Comment:
db layer already adds the creation time to the session object - this causes different times between what you create here and what's stored in db

need to find a way to reuse the same time or add startExecutionTime to session or something like that

Reply:
that's exactly what I was marking by REVIEW. I think we could pass event and session by ref. Then in upper layer (e.g. dispatcher) they will have updated time as well

Reply:
since I'm not relying on session time anymore - I removed all this